docs/core/entity/button.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Button Entity
+sidebar_label: Button
+---
+
+A button entity is an entity that can fire an event / trigger an action towards a device or service but remains stateless from the Home Assistant perspective.
+It can be compared to a real live momentary switch, push-button, or some other form of a stateless switch.
```

Comment:
Maybe clarify that it's not used to integrate real buttons. It's a virtual button inside Home Assistant.

Reply:
Good point, added 👍